spot_driver/src/spot_driver/spot_ros.py

```diff
@@ -480,6 +516,60 @@ def populate_camera_static_transforms(self, image_data):
             self.camera_static_transforms.append(static_tf)
             self.camera_static_transform_broadcaster.sendTransform(self.camera_static_transforms)
 
+    # Arm functions ##################################################
+    def handle_arm_stow(self, srv_data):
+        """ROS service handler to command the arm to stow, home position """
+        resp = self.spot_wrapper.arm_stow()
+        return TriggerResponse(resp[0], resp[1])
+
+    def handle_arm_unstow(self, srv_data):
+        """ROS service handler to command the arm to unstow, joints are all zeros"""
+        resp = self.spot_wrapper.arm_unstow()
+        return TriggerResponse(resp[0], resp[1])
+
+    def handle_arm_joint_move(self, srv_data: ArmJointMovementRequest):
+        """ROS service handler to send joint movement to the arm to execute"""        
+        resp = self.spot_wrapper.arm_joint_move(joint_targets = srv_data.joint_target)
+        return ArmJointMovementResponse(resp[0], resp[1])
+    
+    def handle_force_trajectory(self, srv_data: ArmForceTrajectoryRequest):
+        """ROS service handler to send a force trajectory up or down a vertical force"""
+        resp = self.spot_wrapper.force_trajectory(data=srv_data)
+        return ArmForceTrajectoryResponse(resp[0], resp[1])
+    
+    def handle_gripper_open(self, srv_data):
+        """ROS service handler to open the gripper"""
+        resp = self.spot_wrapper.gripper_open()
+        return TriggerResponse(resp[0], resp[1])
+
+    def handle_gripper_close(self, srv_data):
+        """ROS service handler to close the gripper"""
+        resp = self.spot_wrapper.gripper_close()
+        return TriggerResponse(resp[0], resp[1])    
+    
+    def handle_gripper_angle_open(self, srv_data: GripperAngleMoveRequest):
+        """ROS service handler to open the gripper at an angle"""
+        resp = self.spot_wrapper.gripper_angle_open(gripper_ang = srv_data.gripper_angle)
+        return GripperAngleMoveResponse(resp[0], resp[1])
+    
+    def handle_arm_carry(self, srv_data):
+        """ROS service handler to put arm in carry mode"""
+        resp = self.spot_wrapper.arm_carry()
+        return TriggerResponse(resp[0], resp[1])
+    
+    def handle_body_follow_arm(self, srv_data):
+        """ROS service to send a pose to the end effector"""
```

Comment:
```suggestion
        """ROS service to prepare the arm for body-follow-mode by moving the arm infront of the body, and setting the robot into arm follow mode"""        
```
I read the documentaiton for this  https://dev.bostondynamics.com/python/bosdyn-client/src/bosdyn/client/robot_command.html?highlight=follow_arm_command#bosdyn.client.robot_command.RobotCommandBuilder.follow_arm_command but to be honest I don't quite understand what it does. Does that mean if the next command is a 6DOF command to the arm, the base will run after the arm?

Reply:
I tested it today on our Spot by first calling the /spot/body_follow_hand service and then the /spot/gripper_pose service with a pose outside of the workspace. However the robot did not move. I will therefore delete the /spot/follow_hand service as I do not see why we would need it.